source/config.civet

```diff
@@ -40,7 +40,7 @@ export function findConfig(startDir: string)
 
   return
 
-export function loadConfig(path: string)
+export function loadConfig(path: string): Promise<???>
```

Comment:
This should be a `Promise<CompileOptions>`, with `CompileOptions` from `types/types.d.ts`

Reply:
well i suppose it isnt checked to be a CompileOptions object, but that is how it's used.